locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/DefaultConnectivityResolver.java

```diff
@@ -212,8 +217,10 @@ public ManagementAddressResolveResult resolve(
         }
 
         if (hapChoice == null) {
-            throw new IllegalStateException("jclouds did not return any IP addresses matching " + getNetworkMode() + " in " + location);
+            LOG.error("None of the addresses of node {} are reachable in mode {}", new Object[]{node, getNetworkMode()});
+            throw new IllegalStateException("Could not find any reachable address for node: " + node + " in mode: " + getNetworkMode());
```

Comment:
Strictly speaking Brooklyn didn't look for a "reachable" address if ConnectivityResolverOptions.pollForReachableAddresses was false. How about "Could not determine management address for node..."?